modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportExport.testcase

```diff
@@ -1651,11 +1643,6 @@ definition {
 				key_title = "Accounts Exported",
 				locator1 = "ImportExport#EXECUTION_NOTIFICATION");
 
-			AssertElementPresent(
```

Comment:
Does this mean that we're not going to get a notification when user failed to export? Then we may also need to update the Then task in line 1644? 

> task ("Then the the user should receive a notification for the failed and successful exports") {

Reply:
I think we need to update it because it won't export without selecting any fields, so no failed notification will be shown